fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -123,6 +124,15 @@ private void auditAfterExec(String origStmt, StatementBase parsedStmt, Data.PQue
             .setStmtId(ctx.getStmtId())
             .setQueryId(ctx.queryId() == null ? "NaN" : DebugUtil.printId(ctx.queryId()));
 
+        if (statistics != null && statistics.getNodesStatisticsCount() != 0) {
```

Comment:
I think this can be merge on BE side? Or FE will become the bottleneck

Reply:
Now the calculation will be completed at the be side with the top level fragment, but the collected node statistics will still be transmitted to Fe.